src/recorder/jamrecorder.cpp

```diff
@@ -41,28 +41,40 @@ using namespace recorder;
  * Creates a file for the raw PCM data and sets up a QDataStream to which to write received frames.
  * The data is stored Little Endian.
  */
-CJamClient::CJamClient(const qint64 frame, const int _numChannels, const QString name, const CHostAddress address, const QDir recordBaseDir) :
-    startFrame (frame),
-    numChannels (static_cast<uint16_t>(_numChannels)),
-    name (name),
-    address (address),
-    out (nullptr)
+CJamClient::CJamClient ( const qint64       frame,
```

Comment:
The jam recorder is why I want the formatter.  It was the first large bit of code I did...  Volker almost rejected the whole thing because I wouldn't submit it in house style...  He let me win on the basis it was kept separate.

I've been trying to fix it as I go, when updates happen...  But this just gets most of the work done.